components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/impl/ApisApiServiceImpl.java

```diff
@@ -333,7 +334,14 @@ public Response getCommentOfAPI(String commentId, String apiId, String xWSO2Tena
             Comment comment = apiConsumer.getComment(identifier, commentId);
 
             if (comment != null) {
-                CommentDTO commentDTO = CommentMappingUtil.fromCommentToDTO(comment);
+                CommentDTO commentDTO;
+                if (includeCommenterInfo) {
+                    Map<String, Map<String, String>> userClaimsMap = new HashMap<>();
+                    CommentMappingUtil.retrieveUserClaims(comment.getUser(), userClaimsMap);
```

Comment:
```suggestion
                    Map<String, Map<String, String>> userClaimsMap = CommentMappingUtil.retrieveUserClaims(comment.getUser(), new HashMap<>());
```
Will this be more readable?

Reply:
https://github.com/wso2/carbon-apimgt/pull/9454/commits/dbe6f847730212f0db8456b3628c917d31a75961